apps/web/src/pages/Applications/ApplicantDashboardPage/components/DashboardHeading.tsx

```diff
@@ -240,8 +263,16 @@ const DashboardHeading = ({ user }: DashboardHeadingProps) => {
       )}
       <div
         data-h2-display="base(grid)"
-        data-h2-grid-template-columns="base(100%) p-tablet(repeat(2, minmax(0, 1fr)))"
         data-h2-gap="base(x1)"
+        {...(skillLibraryFlag
+          ? {
+              "data-h2-grid-template-columns":
+                "base(100%) p-tablet(repeat(3, minmax(0, 1fr)))",
```

Comment:
We may want to map this `l-tablet` this is cutting it close 😅 What do you think @substrae ?

![Screenshot 2023-06-23 145449](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/32f6741e-d6b7-4f9e-a62b-ce876961f638)

Reply:
@esizer

Oof, yeah, I think `l-tablet` is a better choice. Good catch!

Reply:
Yeah, good call.  This is minimum size now.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/9496211d-fdb1-4092-98d7-9b67aae7971e)
Ref: [change wrap size](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7089/commits/1ddad25380fcfdc052fa105f03f307b876aeed7b)